### PR TITLE
Fix formatting to fix admonitions in 1.25 docs

### LINKED ISSUE #2695: Note syntax broken in 1.25 docs
Looking at some docs in 1.25 - every note special markdown is blank.  Switching the docs to a 2.x version of the docs fixes it however.

Some examples:

https://docs.jujucharms.com/1.25/en/charms-destroy
https://docs.jujucharms.com/1.25/en/reference-hook-tools

### PR DESCRIPTION
**Pull request by charlesdunbar**
Fixes #2695

### CODE REVIEW

File: src/en/troubleshooting-logs.md [hunk 1]
```diff
@@ -188,17 +188,19 @@ machine in the entire log:
 juju debug-log --replay --include unit-mysql-0 --include machine-1
 ```
 
-!!! Note: The unit can also be written `mysql/0` (as shown by 'juju status').
+!!! Note: 
+	The unit can also be written `mysql/0` (as shown by 'juju status').
```
Comment (pmatulis) [inline src/en/troubleshooting-logs.md]: This one is off.

File: src/en/troubleshooting-logs.md [hunk 2]
```diff
@@ -188,17 +188,19 @@ machine in the entire log:
...
 
 ```bash
 juju debug-log --replay --level WARNING
 ```
 
-!!! Note: The `level` option restricts messages to the specified log-level or
-greater. The levels from lowest to highest are TRACE, DEBUG, INFO, WARNING, and
-ERROR.
+!!! Note: 
+	The `level` option restricts messages to the specified log-level or
```
Comment (pmatulis) [inline src/en/troubleshooting-logs.md]: This one is off.

**Comment by evilnick** [comment]
@pmatulis care to look at this again?

**Comment by pmatulis** [comment]
@charlesdunbar Thank you. And sorry for the delay. :racehorse:

**Pull request merged**